dockerfile-examples/node-22.9.0-yarn.sh

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env bash
 set -e
 #######################################
-# (c) Copyright IBM Corp. 2021
-# (c) Copyright Instana Inc. and contributors 2018
+# (c) Copyright IBM Corp. 2024
```

Comment:
issue: We do not change the copyright header on rename.

Reply:
Reverted